crates/egui/src/memory.rs

```diff
@@ -191,6 +191,10 @@ pub struct Options {
     /// Controls the tessellator.
     pub tessellation_options: epaint::TessellationOptions,
 
+    /// If `true`, This will call `egui::Context::request_repaint()` at the end of each frame
+    /// If `false` (default), egui is only updated if are input events (like mouse movements) or there are some animations in the GUI.
+    pub continuous_mode: bool,
```

Comment:
What is the advantage of this compared to just having the user call `request_repaint`?

Reply:
> What is the advantage of this compared to just having the user call `request_repaint`?

1. Users do not need to worry about request_repaint().
2. This is much more efficient in terms of cput usage, etc.